src/coinbase/staking_reward.ts

```diff
@@ -85,6 +85,7 @@ export class StakingReward {
    * @returns The amount.
    */
   public amount(): Amount {
+    if (this.model.amount == "") return 0;
```

Comment:
I believe this had been discussed previously, but I'm still a bit confused by this `model.amount` special handling 🤔. Do we need this because our backend returns unusual "empty" values instead of simply returning 0?

Reply:
Yes that's correct, @deangalvin-cb raised the same issue on the go sdk

Reply:
Update: Had a nice offline discussion with @facoinbase about this. We'll merge in these changes for now, but then we'll go back and fix this in `c3/cdp-service` and `cloud/staking`